helm/cluster-api-ipam-provider-in-cluster/values.yaml

```diff
@@ -23,12 +23,16 @@ resources:
 
 inClusterIpPool:
   enabled: false
-  name: wc-cp-ips
-  # use either start & end or adresses
-  start: 10.10.222.222
-  end: 10.10.222.229
-  # addresses:
-  # - 10.10.222.223/28
+  pools:
```

Comment:
@jkremser Does it make sense to have these defaults here? They don't seem "sane defaults" to me.

Other than this, I am OK with this PR.

Reply:
right, they are there but not being "activated" :) there is that `.inClusterIpPool.enabled` flag that is false by default and makes everything under `.inClusterIpPool` element ignored. I wanted to have an example there in the default values, but I agree having the real (well not publicly available) ips from ionos there isn't great. So what do you suggest? Should I use some artificial IPs there or comment them out?

Reply:
Ohh. I didn't notice it is not enabled by default. I would use something more generic like "10.0.1.0-10.0.1.255"

Reply:
done, I need to test this first before merging. Also this will break this - https://github.com/giantswarm/mc-bootstrap/blob/main/scripts/setup-config-branch.sh#L554 so we should be careful with releasing. I will open the pr for mc bootstrap once this goes in